### PR TITLE
DSR-166: logging

### PR DESCRIPTION
**Pull request by rupl**
## https://humanitarian.atlassian.net/browse/DSR-166

Logs all info to `/var/log/access.log` — currently set to `VERBOSE`, so it will include stuff like webpack info.

I sanitized the tokens that I found in one type of error. There might be more but our API keys are not shared across environments, so we can deploy to dev and see what happens without endangering our prod API token.

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -17,7 +23,7 @@ module.exports = {
     CTF_HOST: process.env.CTF_HOST,
     CTF_SPACE_ID: process.env.CTF_SPACE_ID,
     CTF_ENVIRONMENT: process.env.CTF_ENVIRONMENT,
-    CTF_CDA_ACCESS_TOKEN: process.env.CTF_CDA_ACCESS_TOKEN, 
+    CTF_CDA_ACCESS_TOKEN: process.env.CTF_CDA_ACCESS_TOKEN,
```
Comment (teodorescuserban) [inline nuxt.config.js]: ⭐️

Comment (rupl) [reply nuxt.config.js]: my editor does all the work for me 😉

**Comment by teodorescuserban** [comment]
Looks good to me, please ping me when the PR is ready to be able to add my review.

File: nuxt.config.js (hunk 1 above)
Comment (teodorescuserban) [reply nuxt.config.js]: :) that would be great, i don't think there is such editor

**Pull request merged**